azure/macos/job.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+  configurationName: ''
+  configurationParameters: ''
+
+jobs:
+  - job: ${{ parameters.configurationName }}
+    pool:
+      vmImage: 'macOS-10.14'
+    steps:
+    - checkout: self
+      fetchDepth: 1
+    - template: brew.yml
+    - script: |
+        export PATH="/usr/local/opt/bison/bin:$PATH"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/openssl@1.1/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/krb5/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/libffi/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/libxml2/lib/pkgconfig"
+        ./buildconf --force
+        ./configure ${{ parameters.configurationParameters }} \
+            --prefix=/usr/local \
+            --enable-phpdbg \
+            --enable-fpm \
+            --with-pdo-mysql=mysqlnd \
+            --with-mysqli=mysqlnd \
+            --with-pgsql \
+            --with-pdo-pgsql \
+            --with-pdo-sqlite \
+            --without-pear \
+            --enable-gd \
+            --with-jpeg \
+            --with-webp \
+            --with-freetype \
+            --enable-exif \
+            --with-zip \
+            --with-zlib \
+            --with-zlib-dir=/usr \
+            --enable-soap \
+            --enable-xmlreader \
+            --with-xsl \
+            --with-tidy=/usr/local/opt/tidyp \
+            --with-xmlrpc \
+            --with-libxml \
+            --enable-sysvsem \
+            --enable-sysvshm \
+            --enable-shmop \
+            --enable-pcntl \
+            --with-readline=/usr/local/opt/readline \
+            --enable-mbstring \
+            --with-curl \
+            --with-gettext==/usr/local/opt/gettext \
+            --enable-sockets \
+            --with-bz2=/usr/local/opt/bzip2 \
+            --with-openssl \
+            --with-gmp=/usr/local/opt/gmp \
+            --with-iconv=/usr/local/opt/libiconv \
+            --enable-bcmath \
+            --enable-calendar \
+            --enable-ftp \
+            --with-pspell=/usr/local/opt/aspell \
+            --with-kerberos \
+            --enable-sysvmsg \
+            --with-ffi \
+            --enable-zend-test \
+            --with-config-file-path=/etc \
+            --with-config-file-scan-dir=/etc/php.d
+      displayName: 'Configure Build'
+    - script: |
+        export PATH="/usr/local/opt/bison/bin:$PATH"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/openssl@1.1/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/krb5/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/libffi/lib/pkgconfig"
+        export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/opt/libxml2/lib/pkgconfig"
```

Comment:
Are these actually necessary or a debugging leftover? It seems very weird that PKG_CONFIG_PATH should be necessary during make.

Reply:
Believe they are left over from debugging, can be removed ... the thing that fixed what that was trying to fix was libxml2 install, I think ...